src/modules/remote-config/useRemoteConfigValue.ts

```diff
@@ -0,0 +1,17 @@
+import { useQuery } from '@tanstack/react-query';
+import { walletPort } from 'src/ui/shared/channels';
+import type { RemoteConfig } from './types';
+
+export function useRemoteConfigValue<K extends keyof RemoteConfig>(key: K) {
+  return useQuery({
+    queryKey: ['wallet/getRemoteConfigValue', key],
+    queryFn: async () => {
+      const value = await walletPort.request('getRemoteConfigValue', { key });
+      return value as RemoteConfig[K];
+    },
+    useErrorBoundary: false,
+    staleTime: 10000,
```

Comment:
Do we need to provide `suspense` here?
Hope this will not lead to ui blinking

Reply:
Yep, let's have `suspense: false`